### PR TITLE
README: Add Install subsection for keeping up to date.

### PR DESCRIPTION
**Pull request by neiljp**
This is for the v0 solution at #**zulip-terminal>Update alerts (README)**, namely to document where updates are announced.

I've not seen an obvious way to subscribe to GH or PyPI updates explicitly, so have left it at text for now.

Feedback welcome.

(v1: Add network call to check API updates & warn if out of date, plus note that we call home)
(v2: Add "advanced" config feature to disable calling home?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ With the python 3.6+ required for running, the following should work on most sys
 
 If you open a different terminal window (or log-off/restart your computer), you'll need to run **step 2** of the above list again before running `zulip-term`, since that activates that virtual environment. You can read more about virtual environments in the [Python 3 library venv documentation](https://docs.python.org/3/library/venv.html).
 
+### Keeping your install up to date
+
+Stable releases are made available on PyPI and GitHub; to ensure you keep up to date with them we suggest checking those sites for updates.
+Stable releases are also announced on the #**announce** stream on the Zulip server https://chat.zulip.org, where you are welcome to make an account.
+
+If running from the `main` git branch, note that this does not automatically update, and you must do so manually.
+This also applies to other source or development installs, including eg. https://aur.archlinux.org/packages/python-zulip-term-git/
```
Comment (neiljp) [inline README.md:80]: ref Arch linux

File: README.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ With the python 3.6+ required for running, the following should work on most sys
 
 If you open a different terminal window (or log-off/restart your computer), you'll need to run **step 2** of the above list again before running `zulip-term`, since that activates that virtual environment. You can read more about virtual environments in the [Python 3 library venv documentation](https://docs.python.org/3/library/venv.html).
 
+### Keeping your install up to date
+
+Stable releases are made available on PyPI and GitHub; to ensure you keep up to date with them we suggest checking those sites for updates.
+Stable releases are also announced on the #**announce** stream on the Zulip server https://chat.zulip.org, where you are welcome to make an account.
```
Comment (neiljp) [inline README.md:77]: Could note about email alerts from that?

**Comment by neiljp** [comment]
@timabbott Was this what you had in mind for v0?

File: README.md (hunk 2 above)
Comment (preetmishra) [reply README.md:77]: Hmm, the email alerts would be stream wide, so I am not sure completely either.

Comment (preetmishra) [reply README.md:77]: We could ask users to set an alert word, say _ZT release_, and adhere to it while posting on the stream.

**Review by preetmishra — COMMENTED** [review]
@neiljp Thanks! Left a comment regarding email notification.

File: README.md (hunk 2 above)
Comment (neiljp) [reply README.md]: I've updated this to mention the expected topic releases will be made in.

**Comment by neiljp** [comment]
@preetmishra Thanks for the thoughts - merging this now after the small update to mention the announce topic as we can iterate from here in various ways once we have a good base :+1:

**Pull request merged**